internal/cri/config/config.go

```diff
@@ -102,6 +102,8 @@ type Runtime struct {
 	// to the runtime spec when the container when PrivilegedWithoutHostDevices is already enabled. Requires
 	// PrivilegedWithoutHostDevices to be enabled. Defaults to false.
 	PrivilegedWithoutHostDevicesAllDevicesAllowed bool `toml:"privileged_without_host_devices_all_devices_allowed" json:"privileged_without_host_devices_all_devices_allowed"`
+	// CgroupWritable enables writable cgroups in non-privileged containers
+	CgroupWritable bool `toml:"cgroup_writable" json:"cgroupWritable"`
```

Comment:
For json, do we want thisCase or this_case?
This has been already inconsistent in the existing config 😞

Reply:
> For json, do we want thisCase or this_case?

I think `camelCase` will be better for new properties going forward, given we don't have a consistent way ourselves yet. Most reference I can find all suggest/require `camelCase`:

1. [google API](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format:~:text=Property%20names%20must%20be%20camel%2Dcased%2C%20ascii%20strings.) suggest camelCase
2. From [a brief search](https://github.com/search?q=repo%3Akubernetes%2Fkubernetes+%22%60json%22&type=code&p=1), kubernetes *mostly* use camelCase 
3. [A `json-api` issue](https://github.com/json-api/json-api/issues/1255) summarizes a few references and all suggest camelCase

Reply:
golang official coding style prefers camel case as well: https://google.github.io/styleguide/go/guide#mixedcaps

Reply:
I agree going forward we should stick to `camelCase`